src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,458 @@
+    # Back in the ./ggsci terminal
+    add extract epos, tranlog, begin now
+    add rmttrail ./dirdat/ab, extract epos megabytes 10
+
+    # This is crucial in order to address the logmining not found error
+    register EXTRACT epos, DATABASE
+
+    # Starts service
+    start epos
+    ~~~
+
+1. Check the status of the Extract by adding a row to `oggadm1.testtable`, committing it to the source, and running the following:
+
+    ~~~
+    # CREATE TABLE IN ORACLE FIRST
+    create table OGGADM1.testtable (col1 number, col2 varchar2(20));
+    alter table OGGADM1.testtable add primary key (col1);
+
+
+    # IN ORACLE
+    INSERT INTO OGGADM1.testtable (col1, col2) VALUES (11, 'Example data');
+    COMMIT;
+
+
+    # IN GGSCI in another terminal
+    GGSCI ({host-name} as oggadm1@ORCL) 28> view param epos
+    EXTRACT epos
+    USERIDALIAS gg_source
+    RMTHOST {host-name}, MGRPORT 8200
+    RMTTRAIL ./dirdat/ab
+    TABLE OGGADM1.testtable;
+    ~~~
+
+    You should see an output similar to the following:
+
+    ~~~
+    Extracting from OGGADM1.TESTTABLE to OGGADM1.TESTTABLE:
+    *** Total statistics since 2023-06-09 19:06:44 ***
+        Total inserts                              1.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           1.00
+    *** Daily statistics since 2023-06-09 19:06:44 ***
+        Total inserts                              1.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           1.00
+    *** Hourly statistics since 2023-06-09 19:06:44 ***
+        Total inserts                              1.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           1.00
+    *** Latest statistics since 2023-06-09 19:06:44 ***
+        Total inserts                              1.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           1.00
+    End of statistics.
+    ~~~
+
+## Set up Replicat
+
+[Replicat](https://docs.oracle.com/goldengate/c1230/gg-winux/GGCON/processes-and-terminology.htm) is an Oracle process that reads trail files and delivers data to a target database.
+
+1. First, make sure you can log into the database from `./ggsci`:
```

Comment:
Make sure to note that this and all other steps in Replicat are on the machine and the directory for `Oracle Golden Gate for PostgreSQL`.